docs/en/user_guides/dataset_prepare.md

```diff
@@ -141,3 +141,37 @@ git clone https://huggingface.co/datasets/liuhaotian/LLaVA-Pretrain --depth=1
    4. TextVQA (textvqa): [train_val_images](https://dl.fbaipublicfiles.com/textvqa/images/train_val_images.zip)
 
    5. VisualGenome (VG): [part1](https://cs.stanford.edu/people/rak248/VG_100K_2/images.zip), [part2](https://cs.stanford.edu/people/rak248/VG_100K_2/images2.zip)
+
+### RefCOCO dataset
+
+#### File structure
+
+```
+
+${DATASET_ROOT}
```

Comment:
XTuner assumes the default `${DATASET_ROOT}` as `./data` in all configs. Thus, recommend to modify it to `./data`